js/apps/admin-ui/src/user-federation/ldap/mappers/LdapMapperDetails.tsx

```diff
@@ -308,7 +309,7 @@ export default function LdapMapperDetails() {
                     variant={SelectVariant.typeahead}
                   >
                     {components.map((c) => (
-                      <SelectOption key={c.id} value={c.id} />
+                      <SelectOption id={c.id} key={c.id} value={c.id} />
```

Comment:
Why does this element need an `id`?

Reply:
Yes, I don't think we need it. I'm going to remove it.